test/unit/math/fwd/prob/lkj_corr_test.cpp

```diff
@@ -1,11 +1,11 @@
 #include <stan/math/fwd.hpp>
 #include <gtest/gtest.h>
-#include <boost/random/mersenne_twister.hpp>
+#include <boost/random/mixmax.hpp>
 #include <boost/math/distributions.hpp>
 
 TEST(ProbDistributionsLkjCorr, fvar_double) {
   using stan::math::fvar;
-  boost::random::mt19937 rng;
+  boost::random::mixmax rng;
```

Comment:
@spinkney another LKJ question for you. This test is now failing on the derivative comparison below:

```cpp
EXPECT_FLOAT_EQ(2.5177896, stan::math::lkj_corr_log(Sigma, eta).d_);
```

Would you know where the fixed value `2.5177896` came from? (I know this wasn't a test you wrote, but I figured you probably understand this better than I do!)

Reply:
No, this doesn't seem right. The derivative is wrt to eta, right? And eta is changing. I think the derivative should be
```r
eta <- 0.3
d <- 4

d_eta <- -(d - 2) * digamma((d - 1) / 2 + eta) + 2 * digamma(d/2 + eta - 1) - 2 * digamma(d + 2 * eta - 2) + log(4)
for (k in 2:(d-1)) {
  d_eta <- d_eta + digamma(eta + (d-1-k)/2)            
}
d_eta
```

Reply:
Using an RNG in this test seems like isn't necessary anyway. There is no randomness used, the value of `eta` is the same every time, we just avoid writing it down explicitly I guess?

If you replace the definition for `eta` with the line ` fvar<double> eta = 1.6294473838061094;`, which is just what the result of the default-constructed mt19937 was giving before, the test passes again.

Reply:
That makes more sense and the rng should be removed.